bundles/automation/org.eclipse.smarthome.automation.module.core.test/ESH-INF/automation/Xrules/ItemRuleDefinition.json

```diff
@@ -8,7 +8,7 @@
     ],
     "configuration": {},
     "description": "Sample Rule for items definition.",
-    "on": [  
+    "triggers": [
```

Comment:
Is this file not used by any test or how comes that no test was failing so far...?